src/Analyzer.PowerShellRuleEngine/Analyzer.PowerShellRuleEngine.csproj

```diff
@@ -8,13 +8,19 @@
   
   <ItemGroup>
     <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.2.4" />
-    <PackageReference Include="Microsoft.PSRule.Rules.Azure" Version="1.18.0-B0010" />
-    <PackageReference Include="Microsoft.PSRule.SDK" Version="2.3.0-B0130" />
+    <PackageReference Include="Microsoft.PSRule.Rules.Azure" Version="1.19.0-B0010" />
```

Comment:
Please update the nuspec file for Analyzer.Core too.  We eventually need a better way to maintain that (created tracking issue #279), but for now, we need to update manually.

Reply:
There are also other out-of-date versions in that nuspec, as well as dependencies that are missing entirely (Bicep).  Can you update those as well?